messaging/src/test/scala/uk/ac/wellcome/messaging/fixtures/SQS.scala

```diff
@@ -100,7 +104,8 @@ trait SQS extends Matchers with Logging {
 
         val arn = getQueueAttribute(
           queueUrl = response.queueUrl(),
-          attributeName = QueueAttributeName.QUEUE_ARN
+          attributeName = QueueAttributeName.QUEUE_ARN,
+          client
```

Comment:
Let's be consistent about keyword arguments:

```suggestion
          client = client
```